src/components/layout.css

```diff
@@ -21,6 +21,15 @@ h3 {
 .markdown {
   @apply text-lg text-gray-900 leading-normal mb-2 font-serif;
 
+  & .videoWrapper{
```

Comment:
_nit_. I'd prefer [kebab-case](https://stackoverflow.com/questions/7560813/why-are-dashes-preferred-for-css-selectors-html-attributes) for CSS classes

Reply:
Done. I just didn't have any examples to follow from since there isn't any custom two word class.